src/Nethermind/Nethermind.Trie/Pruning/IReadOnlyTrieStore.cs

```diff
@@ -17,11 +17,8 @@
 
 namespace Nethermind.Trie.Pruning
 {
-    public static class TrieNodeResolverExtensions
+    public interface IReadOnlyTrieStore : ITrieStore
```

Comment:
bring back the extension method, can stay with an interface but do not move everything to a single class